stl/inc/utility

```diff
@@ -695,6 +695,36 @@ _NODISCARD constexpr bool cmp_greater_equal(const _Ty1 _Left, const _Ty2 _Right)
     return !_STD cmp_less(_Left, _Right);
 }
 
+#ifdef __cpp_lib_concepts
+// STRUCT SYNTH_THREE_WAY
+struct _Synth_three_way {
+    // clang-format off
+    template <class _Ty1, class _Ty2>
+    _NODISCARD constexpr auto operator()(const _Ty1& _Left, const _Ty2& _Right) const requires requires {
+        { _Left < _Right } -> _Boolean_testable;

```

Comment:
Upon looking at this one last time, I think our convention is to wrap `requires requires {` onto the next line. (Still getting used to concepts code...)